### ISSUE TITLE
 Could not resolve property: project.parent.version

### ISSUE THREAD

**Issue by gAmUssA** [joeferner/node-java-maven#5]
Hello,

I'm trying to use node-java-maven with hazelcast and getting following error

```
resolving parent: com.hazelcast:hazelcast-root:3.4.2
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: Could not resolve property: project.parent.version
    at resolveProperty (/poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:339:11)
    at resolveProperty (/poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:336:16)
    at /poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:322:14
    at String.replace (native)
    at resolveSubstitutions (/poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:321:16)
    at resolveDependencyUnknowns (/poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:316:28)
    at /poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:174:11
    at Array.forEach (native)
    at processChildDependencies (/poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:173:27)
    at /poc/hazelcast_nodejs/node_modules/node-java-maven/index.js:158:18
```

package.json

```
{
  "devDependencies": {
    "java": "git://github.com/joeferner/node-java.git"
  },
  "java": {
     "dependencies": [
       {
         "groupId": "com.hazelcast",
         "artifactId": "hazelcast-client",
         "version": "3.4.2"
       }
     ]
   }
}

```

Could you point me where is a problem?

Thank you

**Comment by joeferner** [comment]
Published a new version v0.0.6. Can you give that a try.

**Comment by gAmUssA** [comment]
Thank you @joeferner. Works now https://github.com/gAmUssA/hazelcast_nodejs/blob/master/appWithMaven.js 
p.s. What's better way to execute application code after dependency resolution?

**Comment by gAmUssA** [comment]
@joeferner 

I'm getting errors after I updated to hazelcast 3.5

```
> resolving: com.hazelcast:hazelcast-client:3.5
resolving: com.hazelcast:hazelcast:3.5
resolving parent: com.hazelcast:hazelcast-root:3.5
resolving parent: com.hazelcast:hazelcast-root:3.5
resolving: com.hazelcast:hazelcast-root:3.5
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: Could not resolve property: version
    at resolveProperty (/Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:343:11)
    at resolveProperty (/Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:340:16)
    at /Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:326:14
    at String.replace (native)
    at resolveSubstitutions (/Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:325:16)
    at resolveDependencyUnknowns (/Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:320:28)
    at /Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:174:11
    at Array.forEach (native)
    at processChildDependencies (/Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:173:27)
    at /Users/apple/projects/hazelcast-code-samples/hazelcast-integration/hazelcast-nodejs/node_modules/node-java-maven/index.js:158:18
```

**Comment by joeferner** [comment]
commit: f328b727ab3b76abfdf06edb7bafe384b1b1586a

published v0.0.7 to npm

**Comment by gAmUssA** [comment]
@joeferner That was quick! Thank you!
